### ISSUE TITLE
missing tflite model?

### ISSUE THREAD

**Issue by mtinnes** [ivangrov/TensorFlow-Lite-Age-Gender-Estimation-on-Android#1]
I don't see the tflite model file, or any assets. Is the project complete?

**Comment by ivangrov** [comment]
Hello @mtinnes! I've made a YouTube video (https://youtu.be/ZepT3N6aNFM) about this project and have uploaded the whole Android Studio project to Google Drive: https://drive.google.com/file/d/19Uo30VMudxdOUmRHHn6Z9CVZBn7l4N3f/view?usp=sharing. Links for all of that were in the description of the video, but I've also updated the GitHub page to include them as well. I also share the Keras model through a Google Drive link here: https://drive.google.com/file/d/1K2uH8TVu-kXuw-2oggh-78T4y0-JvgyI/view?usp=sharing . Highly recommend checking the video out for more context 👍

**Comment by mtinnes** [comment]
Ah, got it. Works great thanks! Say we needed different age ranges, would we need to source a different model?

**Comment by ivangrov** [comment]
Correct, you'd need to source/train a different model. I trained this model on [IMDB-WIKI](https://data.vision.ee.ethz.ch/cvl/rrothe/imdb-wiki/) dataset , and I also have a [video](https://youtu.be/o7zHP2UvOlY) about using this dataset on my channel. Since it's a 1 network 2 problems approach (Age & Gender estimation), you could theoretically just leave out the age results, and add a different network computing more age classes. However, I wanna point out that I went with this exact age split because with IMDB-WIKI dataset the data is split quite unevenly and there are, say, very few images of really old or really young people (because it's a celebrity faces dataset), and it's also a scrapped dataset, so not all of the ages and genders there can be 100% correct. I felt like 4 classes served the practical goals I had at the time while also letting me balance the different age classes during training (such that there's roughly the same amount of images for each class through augmentation), and account for some of those inaccuracies in the data if somebody's age is, say, off by a few years.
